tests/getEditorNamespace.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @license Copyright (c) 2003-2020, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import { getEditorNamespace } from '../src';
+import sinon from 'sinon';
+
+const CKEditorNamespace = window.CKEDITOR;
+
+describe( 'getEditorNamespace', () => {
+	const fakeScriptWithNamespace = 'data:text/javascript;base64,d2luZG93LkNLRURJVE9SID0ge307';
+	const fakeScriptWithoutNamespace = 'data:text/javascript;base64,';
+
+	beforeEach( () => {
+		delete window.CKEDITOR;
+	} );
+
+	afterEach( () => {
+		window.CKEDITOR = CKEditorNamespace;
```

Comment:
Can we do
```js
window.CKEDITOR = undefined;
```
and get rid of the `CKEditorNamespace` variable?

Reply:
Yes, preserving `CKEDITOR` namespace is not really important in these tests.